### PR TITLE
Add KerasTransformer example to README

### PR DESCRIPTION
**Pull request by smurching**
See https://github.com/smurching/spark-deep-learning/tree/readme-update to preview the updated README.

I tested the code example in the README locally.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,6 +207,38 @@ Spark DataFrames are a natural construct for applying deep learning models to a
     final_df = transformer.transform(uri_df)
     ```
 
+    #### Other Inputs (1D & 2D)
```
Comment (sueann) [inline README.md]: Tensor Inputs

File: README.md [hunk 2]
```diff
@@ -204,6 +207,38 @@ Spark DataFrames are a natural construct for applying deep learning models to a
...
+
+    `KerasTransformer` applies a Tensorflow-backed Keras model to inputs of up to 2 dimensions. It
```
Comment (sueann) [inline README.md]: nit: TensorFlow

File: README.md [hunk 3]
```diff
@@ -204,6 +207,38 @@ Spark DataFrames are a natural construct for applying deep learning models to a
...
+    loads a Keras model from a given model file path and applies the model to a column of arrays
+    (where an array corresponds to a Tensor), outputting a column of arrays.
+
+    ```python
+    from sparkdl import KerasTransformer
+    from keras.models import Sequential
+    from keras.layers import Dense
+    import numpy as np
+
+    # Generate random input data for our transformer
```
Comment (sueann) [inline README.md]: nit: remove "for our transformer"

File: README.md [hunk 4]
```diff
@@ -204,6 +207,38 @@ Spark DataFrames are a natural construct for applying deep learning models to a
...
+    num_features = 10
+    num_examples = 100
+    input_data = [{"features" : np.random.randn(num_features).tolist()} for i in range(num_examples)]
+    input_df = sqlContext.createDataFrame(input_data)
+
+    # Create and save a single-hidden-layer Keras model for binary classification on our data
```
Comment (sueann) [inline README.md]: nit: remove "on our data"

**Comment by sueann** [comment]
also, @smurching, once this is in do you mind updating & closing the issue https://github.com/databricks/spark-deep-learning/issues/33? thanks!

**Comment by smurching** [comment]
^For sure (re #33), thanks for the feedback!

**Pull request merged**